packages/@okta/vuepress-site/docs/guides/saml-application-setup/configure-saml-in-okta/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Configure SAML in Okta
+---
+To configure the SAML settings of your SAML Service Provider app in Okta:
+
+1. On the **SAML Settings** page, paste the ACS URL that you gathered in the <GuideLink link="../overview">first step</GuideLink> into the **Single sign on URL** field.
+
+2. Paste the Audience URI that you gathered in the <GuideLink link="../overview">first step</GuideLink> into the **Audience URI (SP Entity ID)** field.
+
+    > **Note**: If you are just testing the setup and are using a SAML SP such as this [SAML Service Provider on GitHub](https://github.com/mcguinness/saml-sp), enter this test URL into the **Single sign on URL** and **Audience URI (SP Entity ID)** fields: `http://example.com/saml/sso/example-okta-com`.
+
```

Comment:
Before set the Attribute Statement, it should also set the correct nameId format and value.